dids/src/test/kotlin/web5/sdk/dids/methods/dht/DidDhtTest.kt

```diff
@@ -97,14 +98,51 @@ class DidDhtTest {
       assertNull(did.didDocument!!.services)
     }
 
+    @Test
+    fun `create with bad verification method id throws error`() {
+      val manager = InMemoryKeyManager()
+
+      val exception = assertThrows<IllegalArgumentException> {
+        DidDht.create(
+          manager,
+          CreateDidDhtOptions(
+            verificationMethods = listOf(
+              JsonWebKey2020VerificationMethod(
+                id = "0",
+                publicKeyJwk = manager.getPublicKey(
+                  manager.generatePrivateKey(
+                    JWSAlgorithm.EdDSA,
+                    Curve.Ed25519
+                  )
+                ).toPublicJWK(),
+                relationships = listOf(PublicKeyPurpose.AUTHENTICATION)
+              )
+            ),
+            publish = false
+          )
+        )
```

Comment:
i'm just not sure anyone would be able to figure out how to do this

Reply:
@mistermoe is there a concrete suggestion you have in mind?

This was my best approach towards having options such that:
- The options between ion and Dht are consistent.
- there is a way for consumers of the library to provide the JWK keys they want to be used within the did document. 
- there is a way for consumers to have the key creation handled for them. 

Note that this had been discussed beforehand, when we added options to did ion. It made sense to me back then, and still make sense. So I'd like to understand further what differences you see between ion and dht.